modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -27,12 +28,27 @@ function acquia_cms_common_install($is_syncing) {
       'view media',
     ]);
 
-    // Enable olivero, gin theme.
-    $theme_installer = \Drupal::service('theme_installer');
-    $theme_installer->install(["olivero", "gin"]);
-    $system_theme_config = \Drupal::configFactory()->getEditable('system.theme');
-    $system_theme_config->set('default', 'olivero')->save();
-    $system_theme_config->set('admin', "gin")->save();
+    // Get all available themes.
+    $themesList = \Drupal::service('extension.list.theme')->getList();
+    $themesInstall = ["olivero"];
+    if (isset($themesList['gin'])) {
+      $themesInstall[] = "gin";
+    }
+    // Enable themes.
+    \Drupal::service('theme_installer')->install($themesInstall);
+    $systemThemeConfig = \Drupal::configFactory()->getEditable('system.theme');
+    $systemThemeConfig->set('default', 'olivero')->save();
+    if (in_array("gin", $themesInstall)) {
+      $systemThemeConfig->set('admin', "gin")->save();
+      // Add colors to gin.
+      $ginSettings = \Drupal::configFactory()->getEditable('gin.settings');
+      $ginSettings->set('accent_color', '#096b92');
+      $ginSettings->set('focus_color', '#168ec4');
+      $ginSettings->set('classic_toolbar', 'classic');
```

Comment:
@rajeshreeputra As per the discussion remove this implementation

cc: @chandan-singh7929 @vishalkhode1

Reply:
Done.